curve25519-dalek/src/backend/mod.rs

```diff
@@ -46,14 +46,14 @@ pub mod vector;
 enum BackendKind {
     #[cfg(curve25519_dalek_backend = "simd")]
     Avx2,
-    #[cfg(all(curve25519_dalek_backend = "simd", nightly))]
+    #[cfg(all(curve25519_dalek_backend = "unstable_avx512", nightly))]
     Avx512,
     Serial,
 }
 
 #[inline]
 fn get_selected_backend() -> BackendKind {
-    #[cfg(all(curve25519_dalek_backend = "simd", nightly))]
+    #[cfg(all(curve25519_dalek_backend = "unstable_avx512", nightly))]
```

Comment:
Why is `, nightly` necessary? Isn't it now implicit with the flag? Same for the rest of the file. Actually, it seems like this occurs a lot in the codebase, e.g., in the scalar_mul files below.

Reply:
because nightly is required for unstable features. it's for ensuring this does not compile in stable if someone uses the cfg which would lead to SIGILL